websites/#/アイドルマスターシャイニーカラーズ/presence.ts

```diff
@@ -0,0 +1,134 @@
+const presence = new Presence({
+  clientId: "1103931016525127792",
+}),
+  browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "https://i.imgur.com/ue8EatG.jpg",
+  };
+
+  const { pathname } = window.location;
```

Comment:
This repo uses `document.location` rather than `window.location` 
```suggestion
  const { pathname } = document.location;
```